src/components/renders/bootstrap/select/select-render.tsx

```diff
@@ -79,16 +79,28 @@ export class SelectRender implements Renderer {
         }
       };
 
-      return [
-        <gx-bootstrap />,
-        <select {...attris}>
-          {this.options.map(({ disabled, innerText, selected, value }) => (
-            <option disabled={disabled} selected={selected} value={value}>
-              {innerText}
-            </option>
-          ))}
-        </select>
-      ];
+      return this.component.suggest
+        ? [
+            <input list={this.selectId}></input>,
+
+            <datalist id={this.selectId}>
```

Comment:
I think the `datalist` should have its own id, something like:

```
      const dataListId = `${this.selectId}_datalist`;
```

And then you could have something like this:

```
            <input list={dataListId}></input>,
            <datalist id={dataListId}>
```

Reply:
Yeap, that I asked you in previous comments. I will add that change 😉

Reply:
Setting a variable for the string of `id` and evaluate if `suggest` is enabled and then set it... that way is complex.
I prefer to simply add the "___datalist_" to the `id`. This: `${this.selectId}_datalist`
Checkout the commit f25ad5a to see that simply change